pgmigrate.py

```diff
@@ -184,6 +188,12 @@ def _get_migrations_info_from_dir(base_dir):
                     '\nfirst : %s' % migration.filePath +
                     '\nsecond: %s' % migrations[version].filePath)
             migrations[version] = migration
+    else:
+        LOG.error(
+            "Could not find 'migrations' directory. Did you specify the "
```

Comment:
This message is meaningless, we probably should just raise an exception here.

Reply:
I've removed it